libr/include/r_core.h

```diff
@@ -46,6 +46,7 @@ R_LIB_VERSION_HEADER(r_core);
 #define R_CORE_ANAL_JSON                8
 #define R_CORE_ANAL_KEYVALUE            16
 #define R_CORE_ANAL_JSON_FORMAT_DISASM  32
+#define R_CORE_ANAL_STAR 64
```

Comment:
ident this please

Reply:
Yeah added tabs between 64 and R_CORE_ANAL_STAR